MAUI/DataForm/Editors.md

```diff
@@ -0,0 +1,961 @@
+
+{% tabs %}
+{% highlight MainPage.xaml.cs %}
+
+    dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {   
+
+        if (e.DataFormItem != null && e.DataFormItem.FieldName == "Date" && e.DataFormItem is DataFormDateItem dateItem)
+        {
+            dateItem.Format = "dd, MM, yyyy";
+        }
+    }
+{% endhighlight %}
+{% endtabs %}
+
+`Image`
+
+### Setting MaximumDate and MinimumDate in date editor
+
+You can customize the maximum and minimum allowable dates in the [DatePicker](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/datepicker?view=net-maui-7.0) by setting [MaximumDate]`Link` and [MinimumDate]`Link` in the [DataFormDateItem]`Link` respectively.
+
+{% tabs %}
+{% highlight MainPage.xaml.cs %}
+
+    dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null && e.DataFormItem.FieldName == "Date" && e.DataFormItem is DataFormDateItem dateItem)
+        {
+            dateItem.MinimumDate = new DateTime(2017, 5, 5);
+            dateItem.MaximumDate = new DateTime(2017, 9, 2);
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+`DateMAximumImages`
+
+## Time editor
+
+In the time editor, the [TimePicker](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/timepicker?view=net-maui-7.0&tabs=windows) will be loaded.
+
+
+In [TimePicker](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/timepicker?view=net-maui-7.0&tabs=windows), the default time value (12:00 AM) is displayed by default. You can also adding nullable `TimeSpan` data type for the time picker property in data form, which allows you to set default time and display the (12:00 AM) in time editor. 
+
+{% tabs %}
+{% highlight MainPage.xaml.cs %}
+
+    [DataType(DataType.Time)]
+    [Display(Name = "Birth Time")]
+    public TimeSpan? BirthTime { get; set; }
+
+{% endhighlight %}
+{% endtabs %}
+
+`images`
+
+**Customizing format in time editor**
+
+In the [TimePicker](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/timepicker?view=net-maui-7.0&tabs=windows), short time will be shown by default. You can change the applied format by setting the [Format]`Link`property in [DataFormTimeItem]`Link`.
+
+{% tabs %}
+{% highlight MainPage.xaml.cs %}
+
+    dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null && e.DataFormItem.FieldName == "BirthTime" && e.DataFormItem is DataFormTimeItem timeItem)
+        {
+            timeItem.Format = "HH:mm";
+        }
+    } 
+
+{% endhighlight %}
+{% endtabs %}
+
+`Image`
+
+## CheckBox editor
```

Comment:
Add content for DataFormCheckBoxItem.Color property and include example code on how to change Color using DataFormItemEvent

Reply:
color property added